lib/chat_api/conversations.ex

```diff
@@ -59,6 +63,7 @@ defmodule ChatApi.Conversations do
     end)
   end
 
+  @spec find_by_customer(integer(), integer()) :: [Converation.t()]
```

Comment:
typo (`Converation` -> `Conversation`)

also both of these are UUIDs rather than integers